scripts/rules.matterlint

```diff
@@ -0,0 +1,87 @@
+load "../src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/mode-select-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/network-commissioning-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/power-source-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/power-source-configuration-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/pressure-measurement-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/proxy-configuration-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/proxy-discovery-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/proxy-valid-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/pump-configuration-and-control-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/pwm-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/relative-humidity-measurement-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/scene.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/software-diagnostics-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/switch-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/target-navigator-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/temperature-measurement-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/test-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/thread-network-diagnostics-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/time-format-localization-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/time-synchronization-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/unit-localization-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/user-label-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/wake-on-lan-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml";
+load "../src/app/zap-templates/zcl/data-model/chip/window-covering.xml";
+
+all endpoints {
+    // These attributes follow a different code path and do not have to be
+    // present in the .matter file
+    //
+    // require global attribute generatedCommandList = 65528;
+    // require global attribute acceptedCommandList = 65529;
+    // require global attribute attributeList = 65531;
+
+    require global attribute featureMap = 65532;
+    require global attribute clusterRevision = 65533;
+}
+
+endpoint 0 {
+  // Identifiers for clusters are loaded from XML files
+  require server cluster Basic;
```

Comment:
Could you allow, as a follow-up, cluster IDs, so that even badly named clusters (like "Basic" which is a different cluster for Zigbee than "Basic Information" which is Matter-only) can be stated by ID and we comment on the name?

Reply:
Sure. What would be the syntax?

Originally I thought about `require server cluster 3` but that seemed ugly, so I did the "load name for XML" for user friendlyness. However I can for sure support both numbers and ids.

I was also thinking if IDs should be somehow explicit, like `$Basic` vs `3` to make the parser clear. Coming up with a nice syntax is tedious, willing to accept any and all suggestions.

Reply:
I would like to iterate on updates here, as I imagine we may want to add more validation logic. Will do any changes as followups.